week1/Q1.py

```diff
@@ -0,0 +1,21 @@
+def normalize_word(word):
+	word = word.lower()
+	word = sorted(word)
+	return word
+
+
+def if_permutation(w1, w2):
```

Comment:
The naming of this function seems a bit misleading: is does not check for permutation, and it expect the words to be already normalized. Can we update the implementation to take two arbitrary strings and return True or False whether they are a permutation of each other?